packages/nuxt/src/app/plugins/check-outdated-build.client.ts

```diff
@@ -0,0 +1,21 @@
+import { joinURL } from 'ufo'
+import type { NuxtAppManifestMeta } from '#app'
+import { defineNuxtPlugin, getAppManifest, onNuxtReady, useRuntimeConfig } from '#app'
+
+export default defineNuxtPlugin((nuxtApp) => {
+  let timeout: NodeJS.Timeout
+  const config = useRuntimeConfig()
+
+  async function getLatestManifest () {
+    const currentManifest = await getAppManifest()
+    if (timeout) { clearTimeout(timeout) }
+    timeout = setTimeout(getLatestManifest, 1000 * 60 * 60)
+    const meta = await $fetch<NuxtAppManifestMeta>(joinURL(config.app.cdnURL || config.app.baseURL, '_builds/latest.json'))
```

Comment:
Should we update here too to assetsDir/builds?

Reply:
quite right :+1: